src/routes/index.js

```diff
@@ -1,6 +1,7 @@
 import express from 'express';
 import user from './api/user.routes.js';
 import product from './api/products.routes.js';
+import update from './api/updatePassword.routes';
```

Comment:
Please what do you mean by **update**? please use good name

Reply:
> Please what do you mean by **update**? please use good name

Fixed, thanks